openmc/material.py

```diff
@@ -438,7 +438,7 @@ def add_components(self, components: dict, percent_type: str = 'ao'):
             params['percent_type'] = percent_type
 
             ## check if nuclide
-            if str.isdigit(component[-1]):
+            if not str.isalpha():
```

Comment:
Oops, sorry for leading you astray. This should have been
```suggestion
            if not component.isalpha():
```